controller/backup_controller.go

```diff
@@ -266,15 +266,15 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 
 		if backupTarget.Spec.BackupTargetURL != "" &&
 			backupVolume != nil && backupVolume.DeletionTimestamp == nil {
-			engineClientProxy, backupTargetConfig, err := getBackupTarget(bc.controllerID, backupTarget, bc.ds, log)
+			engineClientProxy, backupTargetClient, err := getBackupTarget(bc.controllerID, backupTarget, bc.ds, log)
```

Comment:
Do we still need `getBackupTarget`? or just use `getBackupTargetClient` instead. I feel we don't need the returned `engineClientProxy` any more here.

Reply:
Fixed.